sched/signal/sig_timedwait.c

```diff
@@ -401,7 +401,7 @@ int nxsig_timedwait(FAR const sigset_t *set, FAR struct siginfo *info,
   siginfo_t unbinfo;
   int ret;
 
-  DEBUGASSERT(set != NULL);
+  DEBUGASSERT(set != NULL && up_interrupt_context() == false);
```

Comment:
but it's removed by https://github.com/apache/nuttx/pull/15377

Reply:
is, but is `nxsig_timedwait` ,it not use sem_wait